cumulus/parachains/integration-tests/emulated/chains/parachains/testing/penpal/src/lib.rs

```diff
@@ -67,5 +68,6 @@ decl_test_parachains! {
 // Penpal implementation
 impl_accounts_helpers_for_parachain!(PenpalA);
 impl_assets_helpers_for_parachain!(PenpalA, Rococo);
+impl_assets_helpers_for_parachain!(PenpalB, Westend);
```

Comment:
I am just thinking if it would be better for clarity to rename:
PenpalA -> PenpalRococo/PenpalForRococo
PenpalB -> PenpalWestend/PenpalForWestend
?
or maybe to split: `chains/parachains/testing/penpal` -> `chains/parachains/testing/penpal-rococo` and `chains/parachains/testing/penpal-westend`? Then it wont fetch both rococo and westend dependency where just one of those is needed.

wdyt?